packages/rancher-gatekeeper/generated-changes/overlay/templates/allowedrepos.yaml

```
@@ -1,3 +1,4 @@
+{{- if .Values.installDefaultConstraintTemplates}}
```

nit: add space to the end of all of these Helm blocks. e.g.

```suggestion
{{- if .Values.installDefaultConstraintTemplates }}
```